TransformationSystem/DB/TransformationDB.py

```diff
@@ -208,9 +229,8 @@ def addTransformation(self, transName, description, longDescription, authorDN, a
 
     ### Add files to the DataFiles table ##################
     catalog = FileCatalog()
-    if addFiles and fileMask:
-      mqDict = json.loads(fileMask)
-      res = catalog.findFilesByMetadata(mqDict)
+    if addFiles and len(inputMetaQuery) > 0:
```

Comment:
No need for `len`

Reply:
Done:
8b0ea690e093080f21879899ec80301b0326996e